examples/src/main/python/sql/datasource.py

```diff
@@ -28,12 +28,25 @@
 
 def generic_file_source_options_example(spark: SparkSession) -> None:
     # $example on:ignore_corrupt_files$
-    # enable ignore corrupt files
+    # enable ignore corrupt files via the data source option
+    # dir1/file3.json is corrupt from parquet's view
+    test_corrupt_df0 = spark.read.option("ignoreCorruptFiles", "true")\
+        .parquet("examples/src/main/resources/dir1/",
+                "examples/src/main/resources/dir1/dir2/")
+    test_corrupt_df0.show()
+    # +-------------+
+    # |         file|
+    # +-------------+
+    # |file1.parquet|
+    # |file2.parquet|
+    # +-------------+
+
+    # enable ignore corrupt files via the configuration
     spark.sql("set spark.sql.files.ignoreCorruptFiles=true")
     # dir1/file3.json is corrupt from parquet's view
-    test_corrupt_df = spark.read.parquet("examples/src/main/resources/dir1/",
+    test_corrupt_df1 = spark.read.parquet("examples/src/main/resources/dir1/",
                                          "examples/src/main/resources/dir1/dir2/")
```

Comment:
Hi, @Yaohua628 . This seems to break the Python Linter CI.
```
./examples/src/main/python/sql/datasource.py:35:17: E128 continuation line under-indented for visual indent
[21](https://github.com/Yaohua628/spark/runs/5967096763?check_suite_focus=true#step:15:21)
                "examples/src/main/resources/dir1/dir2/")
[22](https://github.com/Yaohua628/spark/runs/5967096763?check_suite_focus=true#step:15:22)
                ^
[23](https://github.com/Yaohua628/spark/runs/5967096763?check_suite_focus=true#step:15:23)
./examples/src/main/python/sql/datasource.py:48:42: E128 continuation line under-indented for visual indent
[24](https://github.com/Yaohua628/spark/runs/5967096763?check_suite_focus=true#step:15:24)
                                         "examples/src/main/resources/dir1/dir2/")
```

Reply:
ah yeah, thanks! fixed it

Reply:
Oh, it seems that you are hitting GitHub Action outage.

Reply:
Just tried now, and it is still failing cc: @cloud-fan 
```
Ref: spark-38767
SHA: c02d34370cc71312cf396aaf6550700e5f022cce
Error: There was a new unsynced commit pushed. Please retrigger the workflow.
Error: Unhandled error: Error: There was a new unsynced commit pushed. Please retrigger the workflow.
    at eval (eval at callAsyncFunction (/home/runner/work/_actions/actions/github-script/f05a81df23035049204b043b50c3322045ce7eb3/dist/index.js:6[120](https://github.com/apache/spark/runs/5984516854?check_suite_focus=true#step:2:120):56), <anonymous>:72:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async main (/home/runner/work/_actions/actions/github-script/f05a81df23035049204b043b50c3322045ce7eb3/dist/index.js:6147:20)
```

Reply:
We can check the GA test manually in https://github.com/Yaohua628/spark/actions/workflows/build_and_test.yml

Reply:
Thanks! Seems like all tests passed: https://github.com/Yaohua628/spark/actions/runs/2152938291

Reply:
Thank you for confirmation, @Yaohua628 .